source/sdk/Memory.ooc

```diff
@@ -2,17 +2,8 @@ include string // Memory routines in C are actually in string.h
 version(windows) { include malloc }
 else { include alloca }
 
-// GC_MALLOC zeroes the memory, so in the non-gc version, we prefer to use calloc
-// to the expense of some performance. Don't use malloc instead - existing ooc code
-// is written assuming that gc_malloc zeroes the memory.
-gc_malloc: func (size: SizeT) -> Pointer { gc_calloc(1, size) }
```

Comment:
The `gc_` prefix is outdated and in the long run misleading. The `ooc_` prefix doesn't make for a great name either, especially considering we might not call this language `ooc` for much longer.

I don't see why you can't do your memory debugging. Just change `malloc: extern func (SizeT) -> Pointer` to `_malloc: extern (malloc) func (SizeT) -> Pointer` and then add your own 

```ooc
malloc: func (size: SizeT) -> Pointer { // do whatever }
```